app/classifier/tasks/shortcut.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { Markdown } from 'markdownz';
+
+class Summary extends React.Component {
+  render() {
+    let answer;
+
+    if (this.props.annotation.value) {
+      answer = (
+        <div className="answer">
+          <i className="fa fa-check-circle-o fa-fw" />
+          <Markdown tag="span" inline={true}>{this.props.task.answers[this.props.annotation.value].label}</Markdown>
+        </div>
+      );
+    } else {
+      answer = <div className="answer">No answer</div>;
+    }
+
+    return (
+      <div>
+        <div className="question">
+          {this.props.task.question}
+        </div>
+        <div className="answers">
+          {answer}
+        </div>
+      </div>
+    );
+  }
+}
+
+Summary.propTypes = {
+  annotation: React.PropTypes.shape(
+    { value: React.PropTypes.number }
+  ).isRequired,
+  task: React.PropTypes.shape(
+    {
+      answers: React.PropTypes.array,
+      question: React.PropTypes.string
+    }
+  )
+};
+
+Summary.defaultProps = {
+  annotation: { },
+  task: {
+    answers: [],
+    question: ''
+  }
+};
+
+export default class Shortcut extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      index: null
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.annotation.task !== this.props.annotation.task) {
+      this.setState({ index: null });
+    }
+  }
+
+  toggleShortcut(i, shortcut, e) {
+    if (e.target.checked) {
+      this.props.annotation.shortcut = { index: i };
+      this.setState({ index: i });
+    } else {
+      delete (this.props.annotation.shortcut);
+      this.setState({ index: null });
+      this.props.classification.update('annotations');
```

Comment:
FYI, @eatyourgreens asked a question about the use of the classification resource rather than the onChange handler that other tasks use: https://github.com/zooniverse/Panoptes-Front-End/pull/3522#discussion_r104884244